src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredStylesList = sourceString
+    .split(';')
+    .filter((style) => /\S/.test(style));
```

Comment:
The chained methods `.split(';')` and `.filter((style) => /\S/.test(style))` should have the same indentation level to improve readability, as per checklist item 4.